main.py

```diff
@@ -67,6 +54,12 @@ def parse_args():
         default=512,
         help="Maximum length of generated sequence (prompt+generation)",
     )
+    parser.add_argument(
+        "--num_tasks",
+        type=int,
+        default=None,
+        help="Number of tasks (examples) to solve and evaluate from the benchmark",
+    )
```

Comment:
I'd rename this to `--limit` like in eai/lm-eval or `--samples` to avoid confusion with the tasks arg

Reply:
you're right it can be confusing, `--limit` sounds good